lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/sys.rb

```diff
@@ -263,6 +265,49 @@ def cmd_shell(*args)
     end
   end
 
+  # Drop into an interactive PowerShell session
+  def cmd_powershell(*args)
+    unless client.platform.include?('win')
+      print_error("Platform not supported: #{client.platform}")
+      return
+    end
+
+    path = 'C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe'
```

Comment:
Is that guaranteed to be in the environment regardless of the user the
session is running as?